x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/exceptions_table.tsx

```diff
@@ -0,0 +1,195 @@
+
+    const handleDelete = useCallback((id: string) => () => {}, []);
+
+    const handleExport = useCallback((id: string) => () => {}, []);
+
+    const exceptionsColumns = useMemo((): AllExceptionListsColumns[] => {
+      return getAllExceptionListsColumns(handleExport, handleDelete, history, formatUrl);
+    }, [handleExport, handleDelete, history, formatUrl]);
+
+    const handleRefresh = useCallback((): void => {
+      if (refreshExceptions != null) {
+        setLastUpdated(Date.now());
+        refreshExceptions();
+      }
+    }, [refreshExceptions]);
+
+    useEffect(() => {
+      if (initLoading && !loading && !loadingExceptions && !loadingTableInfo) {
+        setInitLoading(false);
+      }
+    }, [initLoading, loading, loadingExceptions, loadingTableInfo]);
+
+    const emptyPrompt = useMemo((): JSX.Element => {
+      return (
+        <EuiEmptyPrompt
+          title={<h3>{i18n.NO_EXCEPTION_LISTS}</h3>}
+          titleSize="xs"
+          body={i18n.NO_LISTS_BODY}
+        />
+      );
+    }, []);
+
+    const handleSearch = useCallback((search: string) => {
+      const regex = search.split(/\s+(?=([^"]*"[^"]*")*[^"]*$)/);
+      const formattedFilter = regex
+        .filter((c) => c != null)
+        .reduce<ExceptionListFilter>(
+          (filter, term) => {
+            const [qualifier, value] = term.split(':');
+
+            if (qualifier == null) {
+              filter.name = search;
+            } else if (value != null && Object.keys(filter).includes(qualifier)) {
+              return set(qualifier, value, filter);
+            }
+
+            return filter;
+          },
+          { name: null, list_id: null, created_by: null }
+        );
+      setFilters(formattedFilter);
+    }, []);
+
+    const handleSearchChange = useCallback(
+      (event: ChangeEvent<HTMLInputElement>) => {
+        const val = event.target.value;
+        handleSearch(val);
+      },
+      [handleSearch]
+    );
+
+    return (
+      <>
+        <Panel loading={!initLoading && loadingTableInfo} data-test-subj="allRulesPanel">
+          <>
+            {loadingTableInfo && (
+              <EuiProgress
+                data-test-subj="loadingRulesInfoProgress"
+                size="xs"
+                position="absolute"
+                color="accent"
+              />
+            )}
+            <HeaderSection
+              split={false}
+              title={i18n.ALL_EXCEPTIONS}
+              subtitle={<LastUpdatedAt showUpdating={loading} updatedAt={lastUpdated} />}
+            >
+              <EuiFieldSearch
+                data-test-subj="exceptionsHeaderSearch"
+                aria-label={i18n.EXCEPTIONS_LISTS_SEARCH_PLACEHOLDER}
+                placeholder={i18n.EXCEPTIONS_LISTS_SEARCH_PLACEHOLDER}
+                onSearch={handleSearch}
+                onChange={handleSearchChange}
+                disabled={initLoading}
+                incremental={false}
+                fullWidth
```

Comment:
I know it's set to `fullWidth` here but could we make this search box wider? It's one of the things I dislike about the current search bar in the Rules and Monitoring tabs as well. If your rule name is long and your window is small you can't see what you typed.
![image](https://user-images.githubusercontent.com/616158/102221547-4522b200-3eb0-11eb-91fc-4b70583db242.png)

Reply:
Updated! This look better?
<img width="1615" alt="Screen Shot 2020-12-15 at 8 45 56 AM" src="https://user-images.githubusercontent.com/10927944/102222879-0261d980-3eb2-11eb-9f98-e25f022b1932.png">

Reply:
Yes, much better! 🙇‍♂️